vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -265,6 +277,17 @@ void FlyoutShadowNode::AdjustDefaultFlyoutStyle()
   m_flyout.FlyoutPresenterStyle(flyoutStyle);
 }
 
+winrt::Popup FlyoutShadowNode::GetFlyoutParentPopup() const
+{
+  // TODO: Use VisualTreeHelper::GetOpenPopupsFromXamlRoot when running against RS6
+  winrt::Windows::Foundation::Collections::IVectorView<winrt::Popup> popups = winrt::VisualTreeHelper::GetOpenPopups(winrt::Window::Current());
+  for (auto popup : popups)
```

Comment:
This is probably fine but surprised me at first to see a loop.
could be if (popups.Size() > 0) return popups.GetAt(0);     ??

Reply:
Yes, this is leftover from when I thought I was going to be doing some more complicated filtering on the list of poups.  I'll change it up.